content/blogs/newsletter-103.md

```diff
@@ -0,0 +1,132 @@
+---
+title:       "Newsletter 103 - Late 2023 news"
+author:      "GeoB99"
+date:        2023-08-06
+tags:        [ newsletter ]
+---
+
+Salutations from the ReactOS project team!
+On the previous newsletter we talked about the overall ReactOS releasing progress scheme and development status of the project, as well as the hiring of our long-term developer Hermès Bélusca-Maïto (HBelusca).
+We are attempting to make an effort on publishing regularly at least 3 newsletters per year, depending on how the development workflow goes.
+In this newsletter we will highlight some of the contributions by project developers and contributors alike as well as future plans and headlines.
+
+## 0.4.15 release status
+On [102nd newsletter](/blogs/newsletter-102/) we talked about the ReactOS release situation and how we are tackling it.
+Despite our on-going effort and continuation for the project's development, many people have asked about what happened to 0.4.15 and if there will be such a release.
+The answer to these questions is yes -- there will be 0.4.15 and it will be released. The 0.4.15 release has been long delayed due to outstanding regressions and bugs.
+
+Most notably it was a regression which made it impossible [to copy files larger than the amount of RAM using Total Commander 9.12](https://jira.reactos.org/browse/CORE-17624), that was long unfixed for some time after the soft rewrite of the Memory Manager.
+While it has been hackfixed by [PR #5630](https://github.com/reactos/reactos/pull/5630), a proper solution is yet to be worked on. However, this brings us one step closer to releasing 0.4.15.
+
+Due to development resources and manpower constraints we cannot give an exact estimation when 0.4.15 will be released to the general public but we will eventually get there. We are sorry for the inconvenience!
+In the meantime you can try [nightly builds](/getbuilds) that are packed with the latest bugfixes, new features and whatnot but expect also new bugs and such!
+
+## Kernel Debugger overhaul
+Historically, ReactOS was built with the GCC compiler. To help with kernel development, a local kernel debugger (KDBG) was developed. Unfortunately, KDBG could only understand an in-house debug symbol format that was incompatible with Microsoft's PDB format. Because KDBG was a local debugger, it was able to interact with the user via a serial port connection or locally via screen display. Input could be entered via serial or a PS/2 keyboard.
```

Comment:
```suggestion
Historically, ReactOS was built with the GCC compiler.
To help with kernel development, a local kernel debugger (KDBG) was developed.
Unfortunately, KDBG could only understand an in-house debug symbol format that was incompatible with Microsoft's PDB format.
Because KDBG was a local debugger, it was able to interact with the user via a serial port connection or locally via screen display.
Input could be entered via serial or a PS/2 keyboard.
```